src/pretix/control/forms/event.py

```diff
@@ -528,6 +556,36 @@ def __init__(self, *args, **kwargs):
             (a, {"title": a, "data": v}) for a, v in get_fonts().items()
         ]
 
+        # create "virtual" fields for better UX when editing <name>_asked and <name>_required fields
+        self.virtual_keys = []
+        for asked_key in [key for key in self.fields.keys() if key.endswith('_asked') and key[:-5] + 'required' in self.fields]:
+            required_key = asked_key[:-5] + 'required'
+            virtual_key = asked_key + '_required'
+            if virtual_key in self.fields:
+                # there already is a field with virtual_key defined manually, so do not overwrite
+                continue
+
+            asked_field = self.fields[asked_key]
+
+            self.fields[virtual_key] = forms.ChoiceField(
+                label=asked_field.label,
+                help_text=asked_field.help_text,
+                required=True,
+                widget=forms.RadioSelect,
+                choices=[
+                    # default key needs a value other than '' because with '' it would also overwrite even if combi-field is not transmitted
+                    ('do_not_ask', _('Do not ask')),
+                    ('optional', _('Ask, but do not require input')),
+                    ('required', _('Ask and require input'))
+                ]
+            )
+            self.virtual_keys.append(virtual_key)
+
+            if self.initial[required_key]:
+                self.initial[virtual_key] = ['required']
```

Comment:
I haven't tested it, but it seems weird to me that this is a list and not just ``'required'``, have you checked if the string on its own also works?

Reply:
Hm, must have been late. It did not seem to work with just a String – against what I expected, so I changed to a list. But I just checked again and it works with string as well. Sorry!

Reply:
no worries :)